File: 项目开发进度2009年2月9日/SeeMapCell/MapXtreme/6.6/Samples/VisualStudio2005/Web/Features/Thematics/ThematicsVB/App_Code/SampleConstants.vb
Namespace ThematicsWeb

Public Class SampleConstants
    ' eword is a mdb table.
    Public Shared ReadOnly EWorldAlias As String = "eworld"
    ' eworld.tab wrapped eword.mdb table.
    Public Shared ReadOnly EWorldTabFileName As String = "eworld.tab"
    ' ThemeTableAlias is the table on which the sample will create. 
    Public Shared ReadOnly ThemeLayerAlias As String = "world"
    ' ThemeLayerAlias is the FeatureLayer on which the sample will create.
    Public Shared ReadOnly ThemeTableAlias As String = "world"
    ' new LabelLayer alias for this sample.
    Public Shared ReadOnly NewLabelLayerAlias As String = "LabelAliasForBoundData"
    ' alias name for a group layer which contents are going to be changed frenquently.
    Public Shared ReadOnly GroupLayerAlias = "tempGroupLayerAlias"
    ' Bound data columns, these columns come from eworld table.
    Public Shared ReadOnly BoundDataColumns As String() = New String() {"Pop_1994", "Pop_Male", "Pop_Fem"}
    ' Will be used in table's AddColumns method.
    Public Shared ReadOnly SouceMatchColumn As String = "Country"
    ' Will be used in table's AddColumns method.
    Public Shared ReadOnly TableMatchColumn As String = "Country"

    Private Sub New()
    End Sub
End Class

End Namespace

File: 项目开发进度2009年2月9日/SeeMapCell/MapXtreme/6.6/Samples/VisualStudio2005/Web/Features/Thematics/ThematicsCS/App_Code/SampleConstants.cs
using System;

namespace ThematicsWeb
{
	/// <summary>
	/// Summary description for SampleConstants.
	/// </summary>
	public class SampleConstants
	{
		// eword is a mdb table.
		public static string EWorldAlias = "eworld";
		// eworld.tab wrapped eword.mdb table.
		public static string EWorldTabFileName = "eworld.tab";
		// ThemeTableAlias is the table on which the sample will create. 
		public static string ThemeTableAlias = "world";
		// ThemeLayerAlias is the FeatureLayer on which the sample will create.
		public static string ThemeLayerAlias = "world";
		// new LabelLayer alias for this sample.
		public static string NewLabelLayerAlias = "LabelAliasForBoundData";
		// alias name for a group layer which contents are going to be changed frenquently.
		public static string GroupLayerAlias = "tempGroupLayerAlias";
		// Bound data columns, these columns come from eworld table.
		public static string[] BoundDataColumns = new string[]{"Pop_1994", "Pop_Male", "Pop_Fem"};
		// Will be used in table's AddColumns method.
		public static string SouceMatchColumn = "Country";
		// Will be used in table's AddColumns method.
		public static string TableMatchColumn = "Country";

		private SampleConstants(){}
	}
}
